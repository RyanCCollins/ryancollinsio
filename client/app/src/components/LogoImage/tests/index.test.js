import { shallow } from 'enzyme';
import React from 'react';
import { shallowToJson } from 'enzyme-to-json';
import LogoImage from '../index';

describe('<Header />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <LogoImage
        imageSource="https://github.com/RyanCCollins/cdn/blob/master/alumni-webapp/udacity-alumni-png.png?raw=true"
      />,
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
