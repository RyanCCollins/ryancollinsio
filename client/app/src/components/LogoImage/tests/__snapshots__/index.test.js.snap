exports[`<Header /> should render with default props 1`] = `
<styled.div
  animation="fadeIn 1.5s ease-out 0.2s"
  url="https://github.com/RyanCCollins/cdn/blob/master/ryancollinsio-v3/crown.png?raw=true" />
`;
